### ISSUE TITLE
Is there a callback function for successful verification？

### ISSUE THREAD

**Issue by LiangDream** [nuxt-modules/turnstile#251]
### 📚 What are you trying to do?

I don't quite understand the document

**Comment by dargmuesli** [comment]
Yes, there is: https://github.com/nuxt-modules/turnstile/blob/main/src/runtime/types.ts#L25-L28

**Comment by dargmuesli** [comment]
@danielroe what do you think about enabling discussions on this repository to split bug reports and questions?
